.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,48 @@
+---
```

Comment:
Remove this commit, no relation to this branch. Do this in another pull request.